Mathlib/MeasureTheory/MeasurableSpace/CountablyGenerated.lean

```diff
@@ -248,34 +286,25 @@ theorem measurableEquiv_nat_bool_of_countablyGenerated [MeasurableSpace α]
 /-- If a measurable space admits a countable sequence of measurable sets separating points,
 it admits a measurable injection into the Cantor space `ℕ → Bool`
 (equipped with the product sigma algebra). -/
-theorem measurable_injection_nat_bool_of_hasCountableSeparatingOn [MeasurableSpace α]
-    [HasCountableSeparatingOn α MeasurableSet univ] :
-    ∃ f : α → ℕ → Bool, Measurable f ∧ Injective f := by
-  rcases exists_countablyGenerated_le_of_hasCountableSeparatingOn α with ⟨m', _, _, m'le⟩
+theorem measurable_injection_nat_bool_of_countablySeparated [MeasurableSpace α]
+    [CountablySeparated α] : ∃ f : α → ℕ → Bool, Measurable f ∧ Injective f := by
+  rcases exists_countablyGenerated_le_of_countablySeparated α with ⟨m', _, _, m'le⟩
   refine ⟨mapNatBool α, ?_, injective_mapNatBool _⟩
   exact (measurable_mapNatBool _).mono m'le le_rfl
 
 variable {α}
 
 --TODO: Make this an instance
-theorem measurableSingletonClass_of_hasCountableSeparatingOn
-    [MeasurableSpace α] [HasCountableSeparatingOn α MeasurableSet univ] :
+theorem measurableSingletonClass_of_countablySeparated
+    [MeasurableSpace α] [CountablySeparated α] :
     MeasurableSingletonClass α := by
-  rcases measurable_injection_nat_bool_of_hasCountableSeparatingOn α with ⟨f, fmeas, finj⟩
+  rcases measurable_injection_nat_bool_of_countablySeparated α with ⟨f, fmeas, finj⟩
   refine ⟨fun x ↦ ?_⟩
   rw [← finj.preimage_image {x}, image_singleton]
   exact fmeas $ MeasurableSet.singleton _
 
 end SeparatesPoints
 
-instance [MeasurableSpace α] {s : Set α} [h : CountablyGenerated s] [MeasurableSingletonClass s] :
-    HasCountableSeparatingOn α MeasurableSet s := by
```

Comment:
Just checking that I understand the reason for the removal of this instance: that's not needed anymore because the hypotheses imply SeparatesPoints, and CountablyGenerated + SeparatesPoints imply CountablySeparated?

Reply:
Indeed, `inferInstance` is able to prove this. 

This might not have been true previously since `HasCountableSeparatingOn _ MeasurableSet s` is not defeq to `HasCountableSeparatingOn s MeasurableSet univ`